test_runner/regress/test_gc_old_layers.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+@pytest.mark.timeout(10000)
+def test_gc_old_layers(neon_env_builder: NeonEnvBuilder):
+    """
```

Comment:
There are two differences between test_gc_old_layers and test_gc_feedback - last one perform to gc+ompaction iteration after each step and use larger PiTR interval - 10 seconds instead of 1.
Actually the problem (growing ratio between physical and logical size cna be also demonstrated by  test_gc_feedback
so we can remove tst_gc_old_layers and leave only last one.
Or alternativly we can reduce number of iteration of test_gc_oldlaers test and leave it as regression test.
What do you prefer?

Reply:
I removed test_gc_old_layers test